modules/nf-core/bbmap/bbmerge/tests/main.nf.test

```diff
@@ -0,0 +1,63 @@
+nextflow_process {
+
+    name "Test Process BBMAP_BBMERGE"
+    script "../main.nf"
+    process "BBMAP_BBMERGE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "bbmap"
+    tag "bbmap/bbmerge"
+
+    test("sarscov2_illumina_paired - fastq_gz") {
+
+
+        when {
+            process {
+                """
+
+                input[0] = Channel.of([
+                    [ id:'test', single_end:false ], // meta map
+                    [file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true),
+                    file(params.test_data['sarscov2']['illumina']['test_2_fastq_gz'], checkIfExists: true)]
+                    ])
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("sarscov2_illumina_paired - fastq_gz - stub") {
```

Comment:
test says 'fastq_gz' but results are not compressed.

Reply:
I think the test name was more referring to the input file